assets/src/edit-story/components/form/numeric.js

```diff
@@ -83,6 +83,33 @@ function Numeric({
   const [dot, setDot] = useState(false);
   const ref = useRef();
 
+  const handleUpDown = useCallback(
+    ({ key, altKey }) => {
+      if (isMultiple) {
+        return;
+      }
+
+      let newValue;
+      const diff = Big(float && altKey ? 0.1 : 1);
+      if (key === 'ArrowUp') {
+        // Increment value
+        newValue = Big(value).plus(diff);
+      } else if (key === 'ArrowDown') {
+        // Decrement value
+        newValue = Big(value).minus(diff);
+      }
+      onChange(parseFloat(newValue.toString()));
+    },
+    [onChange, value, isMultiple, float]
+  );
+
+  useKeyDownEffect(
+    ref,
+    { key: ['up', 'alt+up', 'down', 'alt+down'], editable: true },
```

Comment:
IMHO `alt` is really uncommon for these now. I don't know why. But I'm typically seeing: 

* No modifier - coarse step
* Shift - fine step

Reply:
I honestly have not strong opinion about either using `alt` or `shift`. As @pbakaus suggested this one, I implemented it like that. A change is easy and can probably be done separately.

Reply:
I modeled this after Chrome DevTools, where `shift` increments in 10, and `alt` increments 0.1. But I'm OK either way.

Reply:
Ok. But we should make it consistent across other actions. The arrows on moveable work with coarse step by default, and shift does the fine step.